planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -2965,7 +2971,22 @@ TurnSignalInfo AvoidanceModule::calcTurnSignalInfo(const ShiftedPath & path) con
       turn_signal_info.turn_signal.command = TurnIndicatorsCommand::ENABLE_RIGHT;
     }
   } else {
-    turn_signal_info.turn_signal.command = TurnIndicatorsCommand::DISABLE;
+    for (const auto & cl : *debug_data_.current_lanelets) {
```

Comment:
I think it is better to use `avoidance_data_.current_lanelets`. `debug_data_` is just for visualization.